frontend/src/views/edit-project/EditProjectView.tsx

```diff
@@ -109,6 +110,11 @@ export const EditProjectView = () => {
     if (displayedRepresentation && displayedRepresentation.id !== representationId) {
       const pathname = generatePath(routeMatch.path, { projectId, representationId: displayedRepresentation.id });
       history.push({ pathname });
+    } else if (displayedRepresentation === undefined) {
```

Comment:
`if (!displayedRepresentation) {` in order to both consider `null` and `undefined`

Reply:
Done